src/Sarif/Visitors/RebaseUriVisitor.cs

```diff
@@ -17,6 +18,7 @@ public class RebaseUriVisitor : SarifRewritingVisitor
     {
         internal const string BaseUriDictionaryName = "originalUriBaseIds";
         internal const string IncorrectlyFormattedDictionarySuffix = ".Old";
+        internal const string OriginalFileKey = "OriginalFileKey";
```

Comment:
>OriginalFileKey [](start = 49, length = 15)

you should initialize this to an instantiated GUID and then you will never risk a collision with an existing property